src/components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import  React, { Component } from 'react'
+import './Form.css'
+
+class Form extends Component {
+  constructor() {
+    super()
+    this.state = {
+      value: ''
+    }
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault()
+    this.props.searchByTitle(this.state.value)
+  }
+
+  handleChange = (event) => {
+    this.setState({value: event.target.value})
+  }
+
+render() {
+    return (
+      <form  className='search-body'onSubmit={this.handleSubmit}>
+        <label>
+          Search by title or enter your minimum desired movie rating:
```

Comment:
I think a better UX could be to make this into 2 separate fields, with the rating coming from a dropdown menu since there's nothing above a 7. (Or at least have some error handling that makes it clear that a 7 is the highest you can go). But think it'll work for now!